diff --git a/jerry.sh b/jerry.sh
index 3ebf999..8bdee7e 100755
--- a/jerry.sh
+++ b/jerry.sh
@@ -24,7 +24,7 @@ images_cache_dir="/tmp/jerry/jerry-images"
 command -v bat >/dev/null 2>&1 && display="bat" || display="less"
 case "$(uname -s)" in
     MINGW* | *Msys) separator=';' && path_thing='' && sed="sed" ;;
-    *arwin) sed="gsed" && player="iina" ;;
+    *arwin) sed="sed" && player="iina" ;;
     *) separator=':' && path_thing="\\" && sed="sed" ;;
 esac
 command -v notify-send >/dev/null 2>&1 && notify="true" || notify="false"
